src/main/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepository.java

```diff
@@ -84,4 +84,9 @@ public EventType mapRow(ResultSet rs, int rowNum) throws SQLException {
                 "SELECT et_event_type_object FROM zn_data.event_type",
                 new EventTypeMapper());
     }
+
+    @Override
+    public void removeEventType(String name) {
+        jdbcTemplate.update("DELETE FROM zn_data.event_type WHERE et_name = ?", name);
```

Comment:
I am no database guy, but I always feel more comfortable having a `LIMIT 1` in my delete.
@CyberDem0n Is that dumb nonsense?